python_ta/debug/accumulation_table.py

```diff
@@ -139,7 +146,11 @@ def _setup_table(self) -> None:
         self._loop_lineno = inspect.getlineno(func_frame) + 1
 
         for_node = get_for_node(func_frame)
-        self.loop_var_name = for_node.target.name
+        if type(for_node.target) == astroid.Tuple:
+            for loop_var in for_node.target.elts:
```

Comment:
I think you should use a comprehension here, to be consistent with the initialization of the loop accumulators.